docs/02.API-REFERENCE.md

```diff
@@ -5523,6 +5531,63 @@ jerry_get_backtrace (uint32_t max_depth);
 
 - [jerry_create_external_function](#jerry_create_external_function)
 
+## jerry_get_api_major_version
+
+**Summary**
+
+Get the major API version.
+
+**Prototype**
+
+```c
+uint32_t
+jerry_get_api_major_version(void);
+```
+
+- return value
+  - API major version
+
+## jerry_get_api_minor_version
+
+**Summary**
+
+Get the minor API version.
+
+**Prototype**
+
+```c
+uint32_t
+jerry_get_api_minor_version(void);
+```
+
+- return value
+  - API minor version
+
+## jerry_get_component_version
+
+**Summary**
+
+Get the component version described in parameter. If the defined components
+asked is not available, then it returns 0, as well as a default value.
```

Comment:
What do you mean by: `..it returns 0, as well as a default value. `? This will return a zero and another value?